### ISSUE TITLE
Telegram Doesn't Show In SysTray

### ISSUE THREAD

**Issue by Alkaris** [telegramdesktop/tdesktop#3751]
I've had this issue for a while, where the Telegram Desktop doesn't properly show in the System Tray, but instead shows a blank spacing where it should be, and still able to access the menu and such. I've tried a number of things to test whether if the same happens if trying a different system tray applet that isn't the default stock applet provided in Linux Mint 18.1 (Cinnamon) 64-bit.
I have tried changing system icons to something else, and this doesn't have any effect on how it appears in the system tray.

### Configuration
**Operating system:** Linux Mint 18.1 (Cinnamon) 64-bit

**Version of Telegram Desktop:** 1.1.19

**Used theme**: Arc-dark

<details><summary><b>Logs</b>:</summary>

[log_03_00.txt](https://github.com/telegramdesktop/tdesktop/files/1194856/log_03_00.txt)
[log_03_15.txt](https://github.com/telegramdesktop/tdesktop/files/1194854/log_03_15.txt)

</details>

- - -

Nothing significant in the logs to tell me why it's not showing the Telegram icon in the system tray. This issue has been persistent since many updates ago, there was some easy workaround fixes in the past to just toggle System Tray Icon ON and OFF again, but that has no longer worked since.

![image](https://user-images.githubusercontent.com/6682988/28887857-d0338e1e-77fd-11e7-8d35-4e4b41ff8103.png)

**Comment by y1n0** [comment]
You're auto starting Telegram, right?
If so, you need to make it to start after the tray loads.
I had the same problem and that solved it.

**Comment by Alkaris** [comment]
@y1n0 - Nope, it's not auto started, it's manually started.

**Comment by JuPlutonic** [comment]
No tray icon at all:(
**Telegram-desktop**: 1.1.22 **OS**: Ubuntu 16.04.3 LTS x64 
**Theme**: Arc-dark 
**Other**: QT5.9.1  Openbox 3.6.1-1ubuntu2 Tint2 0.12.7-2
Started manually by typing in **lxterminal** `/opt/telegram/Telegram`:
[output1.txt](https://github.com/telegramdesktop/tdesktop/files/1275705/output1.txt) that had nothing related tray icon but `fontconfig` related errors. I saw such errors in **Steam**: [opened issue](https://github.com/ValveSoftware/steam-for-linux/issues/3307).
There is no changes when I am starting telegram after I killed it -9 PID/Ctrl-C in terminal (with `Show tray icon` setting turned on before the termination).

**Comment by JuPlutonic** [comment]
[log.txt](https://github.com/telegramdesktop/tdesktop/files/1275761/log.txt)

**Comment by JuPlutonic** [comment]
Tried to resolve absent tray icon by installing sni-qt 0.2.7+16.04.20170217.1-0ubuntu1
Description:  A Qt plugin which turns all QSystemTrayIcon into StatusNotifierItems (appindicators).
Nothing happend

**Comment by IvanGaravito** [comment]
Hi, I tried https://bbs.archlinux.org/viewtopic.php?pid=1683190#p1683190 and worked for me. It says:

> The solution was to disable the option "Enable support for indicators (Requires Cinnamon restart)" in Cinnamon-settings->General. After a Cinnamon restart everything works as expected.

**Comment by JuPlutonic** [comment]
as in 1.1.23: 

- no tray icon

- completely no copy-past
Find out, what tg creates files like qt-trayicon-U20079.png in /tmp
It is only one file that deletes/recreates when number on unread comments changes.

**Comment by JuPlutonic** [comment]
@IvanGaravito 
 Yep, I will search what can provides support for indicators in tint2... [e.g.](http://github.com/dorkbox/SystemTray)
libqt5svg5-dev:amd64 (5.5.1-2build1) - installed but didn't help, libqt5svg5 is already the newest version (5.5.1-2build1) Wil try QT5.9.2, gdb tint2, gdb telegram...

**Comment by Translator5** [comment]
Issue fixed for me with patch 1.2 of the 12. dezember 2017!

**Comment by andreyr** [comment]
Still doesn't work for me. Linux Mint 18.3.    
I had to restart Cinnamon every time after boot.
